lib/cn/cn_metrics.h

```diff
@@ -373,6 +373,36 @@ cn_samp_sub(
     lhs->l_vgarb -= rhs->l_vgarb;
 }
 
+static inline uint
+cn_samp_est(struct cn_samp_stats *s, uint scale)
+{
+    if (s->l_good > 0)
+        return (scale * s->l_alen) / s->l_good;
```

Comment:
I presume alen is a uint64_t. Should you also return a uint64_t from this function?

Reply:
It's just returning a low-res percentage from 0-100 or 0-1000 or some such, so 64-bits seems unnecessary.

Reply:
Would be nice if `pct` was part of the function name then